hw2/ViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ViewController.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class ViewController: UIViewController {
+
+    private let leagueView = LeagueView()
+    //private let matchView = MatchView()
+    
+    private let matchesStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        view.addSubview(leagueView)
+        leagueView.translatesAutoresizingMaskIntoConstraints = false
+        leagueView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(20)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading).offset(16)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing).offset(-16)
```

Comment:
```suggestion
  $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(24)
  $0.leading.trailing.equalToSuperview().inset(16)
```

1. safeAreaLayoutGuide se odražava samo za vertical (npr. gore Dynamic Island, Notch...) horizontaln nema iPhonea s takvim kerefekama :)
2. ovdje možeš koristiti inset umjesto offset. Jer s zapravo s obzirom na superview mi se želimo insetati od njega, prema unutra, što radi inset
3. Možeš chainati leading.trailing
4. top 24 umjesto 20. Idealno je da padding uvijek bude dijeljiv s 8... pa kad već sama stavljaš onda bolje 24